src/systems/systemItemsTable.component.tsx

```diff
@@ -333,15 +327,7 @@ export function SystemItemsTable(props: SystemItemsTableProps) {
       },
       {
         header: 'Usage Status',
-        accessorFn: (row) => {
-          // Assuming row.usage_status contains the numeric value corresponding to the enum
-          const status = Object.values(UsageStatusType).find(
-            (value) =>
-              UsageStatusType[value as keyof typeof UsageStatusType] ===
-              row.item.usage_status
-          );
-          return status || 'Unknown';
-        },
+        accessorFn: (row) => row.item.usage_status,
```

Comment:
```suggestion
       accessorFn:
          type === 'usageStatus' ? undefined : (row) => row.item.usage_status,
```
This disables the column filtering for usage status in the systemsItemDialog. This is needed because the filtered data and the data in the table is going to different. (pre-existing usage status data and inputted usages status data )